docs/source/developers/cpp/windows.rst

```diff
@@ -88,9 +98,16 @@ resolving the build dependencies. This is equivalent to setting
    -DARROW_DEPENDENCY_SOURCE=SYSTEM ^
    -DARROW_PACKAGE_PREFIX=%CONDA_PREFIX%\Library
 
-Note that these packages are only supported for release builds. If you intend
-to use ``-DCMAKE_BUILD_TYPE=debug`` then you must build the packages from
-source.
+To use the Visual Studio IDE with this conda environment activated, launch it by
+running the command:
+
+.. code-block:: shell
+
+   devenv
```

Comment:
```suggestion
To use the Visual Studio IDE with this conda environment activated, launch it by
running the command ``devenv`` in a shell with the development environment
initialized.
```

Reply:
Changed in 67324c11849e0aa02e0bfb22c630705acf67a562. I added the instruction to activate the development environment earlier in this section instead of putting it here.